exercises/practice/complex-numbers/.docs/instructions.append.md

```diff
@@ -0,0 +1,4 @@
+In this exercise you will need to add methods to an existing struct (`Number`).
+You can see an example of this in the Crystal documentation for [getters and setters][getters-and-setters]
+
+[getters-and-setters]: https://crystal-lang.org/reference/1.9/syntax_and_semantics/methods_and_instance_variables.html#getters-and-setters
```

Comment:
Since this isn't a section specifically about reopening/extending classes, I chose to link to a specific version (1.9) so that the link would be less likely to be incorrect in the future.